astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,34 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, raises=False):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
-
+        raises : Bool (default value is False)
+            False : doesn't raise exception if 'name' isn't found. (returns None)
+            True : raises exception if 'name' isn't found.
         Returns
         -------
         coordcls
+        if raises == False:
             The coordinate class corresponding to the ``name`` or `None` if
             no such class exists.
+        if raises == True:
+            The coordinate class corresponding to the ``name`` or raises Exception
+            if no such class exists.
         """
-
-        return self._cached_names.get(name, None)
+        if (raises == False):
```

Comment:
Not need to put the expression under parenthesis. Can you also use ``if not raises:`` for PEP8 reasons*?

*: https://docs.quantifiedcode.com/python-anti-patterns/readability/comparison_to_true.html